app/controllers/dashboard_controller.rb

```diff
@@ -0,0 +1,6 @@
+class DashboardController < ApplicationController
```

Comment:
kindly look into the Namespace and scope topic and chose and use that concept over this 
https://guides.rubyonrails.org/routing.html#controller-namespaces-and-routing
https://medium.com/@malek.stephen44/namespacing-in-rails-4539f969f9e8
https://patrickkarsh.medium.com/namespaces-and-scoping-in-ruby-on-rails-routes-37a17732dca4